### ISSUE TITLE
[jit] Loading JIT'd BatchNorm1d in C++ doesn't work

### ISSUE THREAD

**Issue by suo** [pytorch/pytorch#17327]
Error msg:
```
terminate called after throwing an instance of 'torch::jit::script::ErrorReport'
  what():
Return value was annotated as having type Tuple[] but is actually of type Optional[Tuple[]]:
op_version_set = 0
def _check_input_dim(self,
    input: Tensor) -> Tuple[]:
  _0 = torch.ne(torch.dim(input), 2)
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~...  <--- HERE
  if _0:
    _1 = torch.ne(torch.dim(input), 3)
  else:
    _1 = _0
  if _1:
    ops.prim.RaiseException("Exception")
  else:
    pass
def forward(self,
Aborted (core dumped)
```
Full context [on the forums](https://discuss.pytorch.org/t/loading-batchnorm1d-with-jit/37552)

**Comment by wanchaol** [comment]
I tried to reproduce this, but it works all fine on my side, i will comment on the original

**Comment by wanchaol** [comment]
closing this as it get resolved in the original post
